DESCRIPTION

```diff
@@ -30,6 +30,7 @@ Imports:
     gdxrrw,
     zoo,
     gdxdt,
+    GDPuc,
```

Comment:
GDPuc is now a madrat package and must be loaded under Depends (Use: `GDPuc (>= 1.3.0),`=